front/src/store/modules/serials.store.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+export default {
+  actions: {
+    async getSerials ({ state, commit, rootState }) {
+      await axios
+        .get(`${rootState.baseUrl}/api/Serial`)
+        .then(response => { commit('updateSerials', response.data); });
+    }
+  },
+  mutations: {
+    updateSerials (state, serials) {
+      state.serials = serials;
+    }
+  },
+  state: () => ({
+    serials: []
+  }),
+  getters: {
+    getSerialsList (state) {
+      return state.serials;
```

Comment:
it stills actual - no unnecessary store getters 
please, be more careful to comments and check all, if you don\t agree for some reasons, write about it and we discuss
my opinion was provided in slack https://film-finity.slack.com/archives/C012XRS1R9V/p1592391722006200